src/ex1/StaticClassVisitor.java

```diff
@@ -0,0 +1,90 @@
+package ex1;
+
+import ast.*;
+
+public class StaticClassVisitor implements Visitor {
+    private String className = null;
+
+    public String getResult() {
+        return className;
+    }
+
+    @Override
+    public void visit(IntegerLiteralExpr e) {
+        className = "int";
+    }
+    @Override
+    public void visit(IdentifierExpr e) {
+        className = e.getSymbolTable().varLookup(e.id()).getDecl();
```

Comment:
getDecl gets the name? If so this method should be getVarName or something like that